mitosheet/src/mito/components/toolbar/DataTabContents.tsx

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Mito
+
+
+import React from 'react'
+
+import { ActionEnum, AnalysisData, EditorState, GridState, SheetData, UIState, UserProfile } from '../../types';
+import { MitoAPI } from '../../api/api';
+import { Actions } from '../../utils/actions';
+import ToolbarButton from './ToolbarButton';
+import Dropdown from '../elements/Dropdown';
+import DropdownItem from '../elements/DropdownItem';
+
+export const DataTabContents = (
+    props: {
+        mitoAPI: MitoAPI
+        currStepIdx: number;
+        lastStepIndex: number;
+        highlightPivotTableButton: boolean;
+        highlightAddColButton: boolean;
+        actions: Actions;
+        gridState: GridState;
+        uiState: UIState;
+        setUIState: React.Dispatch<React.SetStateAction<UIState>>;
+        sheetData: SheetData;
+        userProfile: UserProfile;
+        setEditorState: React.Dispatch<React.SetStateAction<EditorState | undefined>>;
+        analysisData: AnalysisData,
+        sheetIndex: number,
+        closeOpenEditingPopups: () => void
+    }): JSX.Element => {
+
+    return (<div className='mito-toolbar-bottom'>
+        <ToolbarButton
+            action={props.actions.buildTimeActions[ActionEnum.Import_Files]}
+            setEditorState={props.setEditorState}
+        />
+        <ToolbarButton
+            action={props.actions.buildTimeActions[ActionEnum.Dataframe_Import]}
+            setEditorState={props.setEditorState}
+        />
+        <ToolbarButton
+            action={props.actions.buildTimeActions[ActionEnum.SNOWFLAKEIMPORT]}
+            setEditorState={props.setEditorState}
+        />
+
```

Comment:
Ah -- one thing that's missing here that a) used to represented in the previous toolbar, and b) is in the mockup for the spec is that we want to get the runtime import actions and display them here. 

This should be pretty easy, you can just use the default import icon for all of them for now (and we can get better at this in the future).